src/mlpack/methods/amf/init_rules/average_init.hpp

```diff
@@ -74,6 +74,77 @@ class AverageInitialization
     H = H + avgV;
   }
 
+  /**
+   * Initialize the matrix W or H to the average value of V with uniform
+   * random noise added.
+   *
+   * @param V Input matrix.
+   * @param r Rank of matrix.
+   * @param M W or H matrix, to be initialized to the average value of V
+   * with uniform random noise added.
```

Comment:
Can you tab this twice (4 spaces) to improve the readability?

Reply:
I believe the original Initialize(V, r, W, H) method only has 3 spaces. Shall I update that as well?

Reply:
Yes, thanks.